metricbeat/mb/module/event.go

```diff
@@ -20,28 +20,19 @@ type EventBuilder struct {
 	FetchDuration time.Duration
 	Event         common.MapStr
 	fetchErr      error
-	filters       *processors.Processors
-	metadata      common.EventMetadata
 }
 
 // Build builds an event from MetricSet data and applies the Module-level
 // filters.
-func (b EventBuilder) Build() (common.MapStr, error) {
+func (b EventBuilder) Build() (beat.Event, error) {
 	// event may be nil when there was an error fetching.
-	event := b.Event
-	if event == nil {
-		event = common.MapStr{} // TODO (akroh): do we want to send an empty event field?
+	fields := b.Event
+	if fields == nil {
+		fields = common.MapStr{} // TODO (akroh): do we want to send an empty event field?
```

Comment:
Could you please remove this comment. This seems to be well established behavior at this point.